### PR TITLE
Illia rudniev/feat/overview page and charts

### PR DESCRIPTION
**Pull request by chesterkmr**
### Description
- Added mocked `Overview page`
- Implemented workflow sorting on FE/BE
- Updated styles of pie chart

### Related issues
 * Provide a link to each related issue.

### Breaking changes
 * Describe the breaking changes that this pull request introduces.

### How these changes were tested
 * Describe the tests that you ran to verify your changes, including devices, operating systems, browsers and versions.

### Examples and references
 * Links, screenshots, and other resources related to this change.

### Checklist
- [] I have read the [contribution guidelines](CONTRIBUTING.md) of this project
- [] I have read the [style guidelines](STYLE_GUIDE.md) of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings and errors
- [] New and existing tests pass locally with my changes

### CODE REVIEW

File: apps/workflows-dashboard/src/common/hooks/useSorting/useSorting.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {
+  getSortingDataFromQuery,
+  SortingData,
+} from '@app/common/hooks/useSorting/helpers/get-sorting-data-from-query';
+import { buildSortingRegex } from '@app/common/hooks/useSorting/utils/build-sorting-regex';
+import { SortingParams } from '@app/common/types/sorting-params.types';
+import { useCallback, useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+export function useSorting(prefix = 'order_by') {
```
Comment (Omri-Levy) [inline apps/workflows-dashboard/src/common/hooks/useSorting/useSorting.ts:10]: In the future please see if there's existing code and I'll make sure to send references. We already have logic for sorting and so on in the backoffice and it uses `qs`. 🙏

File: apps/workflows-dashboard/src/components/molecules/WorkflowsTable/columns.tsx [hunk 2]
```diff
@@ -25,13 +27,13 @@ export const defaultColumns: ColumnDef<IWorkflow>[] = [
         {info.getValue<string>() || ''}
       </div>
     ),
-    header: () => 'Status',
+    header: ({ column }) => <DataTableColumnHeader column={column} title="Status" />,
```
Comment (Omri-Levy) [inline apps/workflows-dashboard/src/components/molecules/WorkflowsTable/columns.tsx:30]: Good chance you can get `title` from `column`.

File: apps/workflows-dashboard/src/components/atoms/PieChart/utils/calculateChartDataSum.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { PieChartData } from '@app/components/atoms/PieChart/types';
+
+export function calculateChartDataSum(chartData: PieChartData[]) {
```
Comment (Omri-Levy) [inline apps/workflows-dashboard/src/components/atoms/PieChart/utils/calculateChartDataSum.ts:3]: Could be changed to a general sum helper.

File: apps/workflows-dashboard/src/components/molecules/Header/header-navigation-links.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { NavigationLink } from '@app/components/molecules/Navigation';
+
+export const headerNavigationLinks: NavigationLink[] = [
```
Comment (Omri-Levy) [inline apps/workflows-dashboard/src/components/molecules/Header/header-navigation-links.ts:3]: Check out TypeScript's `satisfies` and `as const`.

File: apps/workflows-dashboard/src/pages/Workflows/Workflows.tsx [hunk 5]
```diff
@@ -4,18 +4,24 @@ import { FilterComponent } from '@app/pages/Workflows/components/organisms/Workf
...
+import { useSorting } from '@app/common/hooks/useSorting';
 
 const filterComponents: FilterComponent[] = [StatusFilterComponent];
 
 export const Workflows = () => {
   const { filters, setFilters } = useWorkflowsFilters();
-  const { data, isLoading, isFetching } = useWorkflows(filters);
-  const { data: metric, isLoading: isLoadingMetric } = useWorkflowsMetric();
+  const { sortingKey, sortingDirection } = useSorting('order_by');
+  const { data, isLoading, isFetching } = useWorkflows(
```
Comment (Omri-Levy) [inline apps/workflows-dashboard/src/pages/Workflows/Workflows.tsx:19]: Make sure we're using the intended option between `isFetching` and `isLoading`.

File: apps/workflows-dashboard/src/pages/Workflows/components/layouts/WorkflowsLayout/Header.tsx [hunk 6]
```diff
@@ -3,5 +3,5 @@ interface Props {
 }
 
 export const Header = ({ children }: Props) => {
-  return <header className="flex flex-col gap-4 p-4">{children}</header>;
+  return <div className="flex flex-col gap-4">{children}</div>;
```
Comment (Omri-Levy) [inline apps/workflows-dashboard/src/pages/Workflows/components/layouts/WorkflowsLayout/Header.tsx:6]: Is there a more semantic element to use here?

**Review by alonp99 — APPROVED** [review]

**Pull request merged**